NadekoBot.Core/Modules/Gambling/Common/Blackjack/Blackjack.cs

```diff
@@ -229,21 +229,33 @@ AA 5
                     if (usr.State == User.UserState.Blackjack)
                         usr.State = User.UserState.Won;
                     else if (usr.State == User.UserState.Stand)
-                        usr.State = hw < usr.GetHandValue()
-                            ? User.UserState.Won
-                            : User.UserState.Lost;
+                        if (hw == usr.GetHandValue())
+                            usr.State = User.UserState.Tie;
+                        else if (hw < usr.GetHandValue())
+                            usr.State = User.UserState.Won;
                     else
                         usr.State = User.UserState.Lost;
+
                 }
             }
 
             foreach (var usr in Players)
             {
-                if (usr.State == User.UserState.Won || usr.State == User.UserState.Blackjack)
+                if (usr.State == User.UserState.Won)
```

Comment:
Use a switch statement https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/statements/selection-statements#the-switch-statement